### DISCUSSION TITLE
Add regex option to password policies

### DISCUSSION THREAD

**Discussion by hacheraw** [dani-garcia/vaultwarden#3459] [Ideas]
As the title says. Add regex option to password policies (both Master and generator)

I'm asking this because I'm okay with special characters `.-_` and I'd like to allow them to count as special, but with the current options, my users are forced to add any of `!@#$%^&*` even if their passwords already have any of `.-_`

PS: I've done a search in discussions but it throwed 17 pages... (only looked the first 3 tbh)

**Comment by BlackDex** [comment]
This is not a feature this project can add. It's a feature that needs to be added by Bitwarden since all clients need to support this.

https://community.bitwarden.com/c/feature-requests/5/none
